packages/cozy-clisk/docs/RequestInterceptor.md

```diff
@@ -0,0 +1,77 @@
+# RequestInterceptor
+
+## Intercepting HTTP requests in clisk konnectors
+
+To intercept HTTP requests, you can init the RequestInterceptor like this :
+
+```javascript
+import {
+  ContentScript,
+  RequestInterceptor
+} from 'cozy-clisk/dist/contentscript'
+
+const requestInterceptor = new RequestInterceptor([
+  {
+    label: 'testRequest',
+    method: 'GET',
+    url: 'test/api',
+    serialization: 'json'
+  }
+])
+requestInterceptor.init()
+
+const connector = new ContentScript({ requestInterceptor })
+```
+
+This will intercept all requests with method 'GET' and containing 'test/api' in their url. The
```

Comment:
so this is `*test/api*` right?

Reply:
I chose to make these stars implicit, you think we should accept glob patterns or regexp ?

Reply:
I don't like the implicit, and I think we should avoid it. I don't know what we should accept, but regex seems a good candidate.